pkgs/development/libraries/ucx/default.nix

```
@@ -1,7 +1,8 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, doxygen
-, numactl, rdma-core, libbfd, libiberty, perl, zlib, symlinkJoin
+{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, doxygen, numactl
+, rdma-core, libbfd, libiberty, perl, zlib, symlinkJoin, pkgconfig
```

```suggestion
, rdma-core, libbfd, libiberty, perl, zlib, symlinkJoin, pkg-config
```

I got this error:

```
error: Function called without required argument "pkgconfig" at /home/gabriel/.cache/nixpkgs-review/pr-221347/nixpkgs/pkgs/development/libraries/ucx/default.nix:2, did you mean "pkg-config" or "pkgconf"?
(use '--show-trace' to show detailed location information)
nix --experimental-features nix-command --system x86_64-linux eval --json --impure --no-allow-import-from-derivation --expr (import /nix/store/f6qs6s51w6prb5lpz4klk85rywn4p1jb-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/nix/evalAttrs.nix { allowAliases = false; attr-json = /tmp/nix-shell.tbsz32/tmpvbui2i38; }) failed to run, /tmp/nix-shell.tbsz32/tmpvbui2i38 was stored inspection
```

`pkgconfig` is a Python module. Did you perhaps mean to import `pkg-config`?

---

Yes, you're right!

---

this still needs to be fixed